src/components/shared/TreeView.js

```diff
@@ -572,7 +580,7 @@ TreeView.propTypes = {
   selectedNodeId: PropTypes.number,
   onExpandedNodesChange: PropTypes.func.isRequired,
   onSelectionChange: PropTypes.func.isRequired,
-  highlightString: PropTypes.string,
+  highlightStrings: PropTypes.arrayOf(PropTypes.string),
```

Comment:
I kept the proptypes here because it's still used in the marker table which isn't flow checked yet.